### PR TITLE
Partner Manager Edit Endpoint Feature

### PR DESCRIPTION
**Pull request by dhirsch1**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+Although endpoint settings don't often require changes, you might need to modify them under certain circumstances. For example:
+
+* Customers move their data center, so you must change the IP address or host name.
+* Customers update their credentials, so you must change the user credentials in the endpoint configuration.
+* Customers change their file naming conventions and response timeout requirements, so you must change these configurations for the endpoint.
+* The downstream API that receives translated inbound data moves from one region to another, so you must change the endpoint URL.
+
+To modify an endpoint:
+
+. Access the endpoint you want to modify from the endpoint owner's page (either the *<host>* or *<partner>* page, depending on the endpoint).
```
Comment (vijayang) [inline modules/ROOT/pages/modify-endpoints.adoc]: Change to: "host, partner or 3rd party page"

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+If required, Partner Manager also automatically deploys the Mule runtime applications that Partner Manager uses. The deployment of Mule runtime applications can take at five minutes or longer to complete, depending on how many runtime applications are affected. 
+
+[[as2-receive]]
+== AS2 Receive Endpoints
+
+You can modify the following fields for an AS2 Receive from Partners endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
```
Comment (vijayang) [inline modules/ROOT/pages/modify-endpoints.adoc]: Change "Requires the Deployment of Runtime Applications" to "Results in the redeployment of runtime application for the endpoint" here and in subsequent sections.

"Requires the Deployment of Runtime Applications" may make users to think they will need to redeploy the app themselves. The redeployment happens automatically when the "Deploy" button is clicked.

Clicking the "Deploy" button however necessary regardless of the field modified

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+You can modify the following fields for an FTP and SFTP Send to Partners or Target at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Description |No
+|Endpoint name |No
+|File name pattern |No
+|Host |No
+|Password |No
```
Comment (vijayang) [inline modules/ROOT/pages/modify-endpoints.adoc]: Please change to "Yes"

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+|Port |No
+|Target path |No
+|Username |Yes
```
Comment (vijayang) [inline modules/ROOT/pages/modify-endpoints.adoc:113]: Please change to "Yes"

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+
+You can modify the following fields for an HTTPS Send to Partners or Target at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Authentication type and associated fields
+ a| 
+ * No, for the Authentication type field
+ * Yes for fields associated with the authentication type
```
Comment (vijayang) [inline modules/ROOT/pages/modify-endpoints.adoc]: It should be just "Yes" - with no bullets. 
When Authentication type is changed, associated fields also change.

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+[%header%autowidth.spread]
+|===
+|Field |Requires the Deployment of Runtime Applications
+|Authentication type and associated fields | Yes
+|Connection idle timeout (in milleseconds) | Yes
+|Credentials | Yes
+|Description | Yes
+|Endpoint name | Yes
+|Manage with API Manager and API Auto discovery id | Yes, if the *Private port (when deploying to VPC)* field is enabled
```
Comment (vijayang) [inline modules/ROOT/pages/modify-endpoints.adoc]: This should be just "Yes". Private port config does not affect this behavior.

**Review by vijayang — DISMISSED** [review]

File: modules/ROOT/pages/modify-endpoints.adoc (hunk 2 above)
Comment (valkyrie69) [reply modules/ROOT/pages/modify-endpoints.adoc]: @vijayang Can we say "Redeploys the runtime application for the endpoint" instead?

File: modules/ROOT/pages/modify-endpoints.adoc (hunk 3 above)
Comment (valkyrie69) [reply modules/ROOT/pages/modify-endpoints.adoc]: @vijayang Which one should be yes? All of them?

File: modules/ROOT/pages/modify-endpoints.adoc (hunk 4 above)
Comment (valkyrie69) [reply modules/ROOT/pages/modify-endpoints.adoc:113]: @vijayang Same question as above.

File: modules/ROOT/pages/create-endpoint.adoc [hunk 7]
```diff
@@ -1,6 +1,6 @@
 = Creating Endpoints
 
-You can create new endpoints from the *Partners* or *Host* page, or by using the Message Flow wizard.
+You can create new endpoints from the *Partners* or *<host>* page, or by using the Message Flow wizard.
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-endpoint.adoc]: @vijayang Should I also add third-party page to this one?

File: modules/ROOT/pages/modify-endpoints.adoc (hunk 2 above)
Comment (vijayang) [reply modules/ROOT/pages/modify-endpoints.adoc]: @valkyrie69 - Yes! that will be perfect.

File: modules/ROOT/pages/modify-endpoints.adoc (hunk 3 above)
Comment (raragon-mule) [reply modules/ROOT/pages/modify-endpoints.adoc]: SFTP/FTP Send - user and password should be yes.
SFTP/FTP Receive - Endpoint name should be yes.

File: modules/ROOT/pages/modify-endpoints.adoc (hunk 4 above)
Comment (raragon-mule) [reply modules/ROOT/pages/modify-endpoints.adoc:113]: This is fine, no changes needed based on this:

SFTP/FTP Send - user and password should be yes.
SFTP/FTP Receive - Endpoint name should be yes.

File: modules/ROOT/pages/create-endpoint.adoc (hunk 7 above)
Comment (raragon-mule) [reply modules/ROOT/pages/create-endpoint.adoc]: yes!

**Review by pmantellini — DISMISSED** [review]
LGTM

File: modules/ROOT/pages/delete-endpoints.adoc [hunk 8]
```diff
@@ -14,6 +14,8 @@ To locate endpoints that don’t have associated message flows, look for `-` in
 
 == See Also
 
-* xref:endpoints.adoc[Endpoint Types and Protocols]
-* xref:create-outbound-message-flow.adoc[Create Outbound Message Flows]
-* xref:configure-message-flows.adoc[Create Inbound Message Flows]
+* xref:endpoints.adoc[Endpoints]
+* xref:create-endpoints[Creating Endpoints]
```
Comment (sathya0) [inline modules/ROOT/pages/delete-endpoints.adoc]:
```suggestion
* xref:create-endpoint.adoc[Creating Endpoints]
```

File: modules/ROOT/pages/delete-endpoints.adoc [hunk 9]
```diff
@@ -14,6 +14,8 @@ To locate endpoints that don’t have associated message flows, look for `-` in
...
+* xref:modify-endpoints[Modifying Endpoints]
```
Comment (sathya0) [inline modules/ROOT/pages/delete-endpoints.adoc]:
```suggestion
* xref:modify-endpoints.adoc[Modifying Endpoints]
```

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 10]
```diff
@@ -0,0 +1,182 @@
...
+|Description | No
+|Endpoint name | No
+|Response timeout (in milliseconds) | No
+|URL | No
+|===
+
+== See Also
+
+* xref:endpoints.adoc[Endpoints]
+* xref:create-endpoints.adoc[Creating Endpoints]
```
Comment (sathya0) [inline modules/ROOT/pages/modify-endpoints.adoc]:
```suggestion
* xref:create-endpoint.adoc[Creating Endpoints]
```

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 11]
```diff
@@ -0,0 +1,182 @@
...
+* xref:deleting-endpoints.adoc[Deleting Endpoints]
```
Comment (sathya0) [inline modules/ROOT/pages/modify-endpoints.adoc]:
```suggestion
* xref:delete-endpoints.adoc[Deleting Endpoints]
```

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 12]
```diff
@@ -0,0 +1,182 @@
...
+
+[[http-receive]]
+== HTTP Receive Endpoints
+
+You can modify the following fields for an HTTP Receive from Partners or Source at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Redeploys the Runtime Application for the Endpoint
+|Connection idle timeout (in milleseconds) | Yes
```
Comment (sathya0) [inline modules/ROOT/pages/modify-endpoints.adoc]:
```suggestion
|Connection idle timeout (in milliseconds) | Yes
```

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 13]
```diff
@@ -0,0 +1,182 @@
...
+[[https-receive]]
+== HTTPS Receive Endpoints
+
+You can modify the following fields for an HTTPS Receive from Partners or Source at Host endpoint:
+
+[%header%autowidth.spread]
+|===
+|Field |Redeploys the Runtime Application for the Endpoint
+|Authentication type and associated fields | Yes
+|Connection idle timeout (in milleseconds) | Yes
```
Comment (sathya0) [inline modules/ROOT/pages/modify-endpoints.adoc]:
```suggestion
|Connection idle timeout (in milliseconds) | Yes
```

File: modules/ROOT/pages/modify-endpoints.adoc [hunk 14]
```diff
@@ -0,0 +1,182 @@
+== Modifying Endpoints
```
Comment (sathya0) [inline modules/ROOT/pages/modify-endpoints.adoc]:
```suggestion
= Modifying Endpoints
```

**Review by sathya0 — APPROVED** [review]

**Pull request merged**